Gopkg.toml

```diff
@@ -38,7 +38,7 @@ required = [
   name = "github.com/docker/distribution"
 
 [[constraint]]
-  branch = "master"
+  branch = "remove-status-heartbeat"
```

Comment:
Revert & refresh after merging this: https://github.com/giantswarm/apiextensions/pull/206